saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/metadata/OpenSamlMetadataResolver.java

```diff
@@ -238,6 +244,15 @@ private String serialize(EntitiesDescriptor entities) {
 		}
 	}
 
+	/**
+	 * Configure whether to sign the metadata.
+	 *
+	 * @since 6.3
```

Comment:
Now that we are in the RC phase, this will go into the 6.4 release instead. Please change the `@since` attribute to reflect this.